### PR TITLE
Leverage g8os sal during vdisk/vm creation

### PR DESCRIPTION
**Pull request by grimpy**
Signed-off-by: Jo De Boeck <deboeck.jo@gmail.com>

### CODE REVIEW

**Comment by grimpy** [comment]
Please have a look at the idea, I still need to test this

File: templates/vm/actions.py [hunk 1]
```diff
@@ -13,20 +13,8 @@ def input(job):
 
 
 def get_node_client(service):
```
Comment (zaibon) [inline templates/vm/actions.py]: let's rename this to get node. so later in the code we avoid having `client.client.kvm...` and have instead `node.client.kvm.`

File: templates/vm/actions.py [hunk 2]
```diff
@@ -95,24 +84,23 @@ def init(job):
...
     for nbdserver in service.producers.get('nbdserver', []):
         # build full path of the nbdserver unix socket on the host filesystem
-        container = nbdserver.parent
-        if container.model.data.id == 0:
+        container = Container.from_ays(nbdserver.parent)
+        if not container.is_running():
             # start container
-            j.tools.async.wrappers.sync(container.executeAction('start'))
-        container_root = get_container_root(service, container.model.data.id)
+            j.tools.async.wrappers.sync(nbdserver.parent.executeAction('start'))
```
Comment (zaibon) [inline templates/vm/actions.py]: why not using the sal object and do `container.start()` ?

File: templates/vm/actions.py (hunk 1 above)
Comment (grimpy) [inline templates/vm/actions.py]: Sounds good!

File: templates/vm/actions.py (hunk 2 above)
Comment (grimpy) [inline templates/vm/actions.py]: to make sure the state in the ays model is set properly aswell

**Pull request merged**